Vagrantfile

```diff
@@ -6,72 +6,23 @@
 # backwards compatibility). Please don't change it unless you know what
 # you're doing.
 
+$script = <<SCRIPT
+cd /home/vagrant/oppia
+apt-get update && apt-get install -y python-pip python-dev unzip
+bash ./scripts/setup.sh
+bash ./scripts/install_third_party.sh
+SCRIPT
```

Comment:
So on your recommendation, I changed the inline script to the following: 
```vagrant
$script = <<SCRIPT
cd /home/vagrant/oppia
apt-get update
bash ./scripts/start.sh
SCRIPT
```

The build failed for lack of an `unzip` package. After adding unzip, it failed because of a lack of pip. After adding pip, it failed to install numpy because it wasn't using python-dev.